docs/serving/autoscaling/README.md

```diff
@@ -0,0 +1,18 @@
+One of the main features of Knative is automatic scaling of replicas for an application to closely match incoming demand, including scaling applications to zero if no traffic is being received.
+Knative Serving enables this by default, using the Knative Pod Autoscaler (KPA).
+The Autoscaler component watches traffic flow to the application, and scales replicas up or down based on configured metrics.
+
+Knative services default to using autoscaling settings that are suitable for the majority of use cases. However, some workloads may require a custom, more finely-tuned configuration.
+This guide provides information about configuration options that you can modify to fit the requirements of your workload.
+
+For more information about how autoscaling for Knative works, see the [Autoscaling concepts](./autoscaling-concepts.md) documentation.
+
+For more information about which metrics can be used to control the Autoscaler, see the [metrics](./autoscaling-metrics.md) documentation.
+
+## Optional autoscaling configuration tasks
+
+* Configure your Knative deployment to use the Kubernetes [Horizontal Pod Autoscaler (HPA)](../../install/any-kubernetes-cluster.md#optional-serving-extensions) instead of the default KPA.
+* Disable scale to zero functionality for your cluster ([global configuration only](./scale-to-zero.md)).
+* Configure the [type of metrics](./autoscaling-metrics.md) your Autoscaler consumes.
+* Configure [concurrency limits](./concurrency.md) for applications.
+* Try out the [Go Autoscale Sample App](./autoscale-go/README.md).
```

Comment:
Do you see doing a similar move for other samples in the future (e.g. traffic splitting samples into the networking subsection)?

Reply:
Yep, I'd like to do this with all of them, it just makes more sense to me as a place where someone would look for it. WDYT?

Reply:
I don't think we've discussed it; I don't have an intrinsic problem with it, but it would be good to have a documented strategy.